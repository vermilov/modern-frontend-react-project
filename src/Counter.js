import React from "react";

export const Counter = ({ counter }) => (
    <span>
        <p>{counter}</p>
    </span>
);

export default Counter;
