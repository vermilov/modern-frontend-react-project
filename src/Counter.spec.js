import React from "react";
import renderer from "react-test-renderer";
import { Counter } from "./Counter";

describe("Counter tests", () => {
    test("Snapshot renders", () => {
        const component = renderer.create(<Counter counter={1} />);
        const tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
