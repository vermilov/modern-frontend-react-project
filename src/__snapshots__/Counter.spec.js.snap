// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter tests Snapshot renders 1`] = `
<span>
  <p>
    1
  </p>
</span>
`;
